resource/resource.go

```diff
@@ -50,7 +52,12 @@ var (
 		"log_level":      {},
 		"log_url":        {},
 		"context_prefix": {},
+		//
+		"gchat_webhook": {},
 	}
+
+	// States that will trigger a chat notification.
+	statesToNotifyChat = []string{"error", "failure"}
```

Comment:
But I would make `success` configurable/optional. There are pipelines (quite a few pipeline actually, but not enough to justify doing it everywhere) where it is wanted to configure also the success one.

Example: when we build Docker image in linux image build. I saw in a lot of channels also successful pertracker job registrations...

Reply:
This is why I had by default also success and pending.

I want to start by keeping the changes as little as possible, this a new feature and we don't even know if having a resource doing two things works in production.

In this particular case, I can see the rationale for making this setting customizable, but we are not there yet.